turbinia/task_manager.py

```diff
@@ -190,16 +192,21 @@ def add_evidence(self, evidence_):
           'Jobs must be registered before evidence can be added')
     log.info('Adding new evidence: {0:s}'.format(str(evidence_)))
     job_count = 0
-    jobs_allowlist = evidence_.config.get('jobs_allowlist', [])
-    jobs_denylist = evidence_.config.get('jobs_denylist', [])
-    if jobs_denylist or jobs_allowlist:
-      log.info(
-          'Filtering Jobs with allowlist {0!s} and denylist {1!s}'.format(
-              jobs_allowlist, jobs_denylist))
-      jobs_list = jobs_manager.JobsManager.FilterJobObjects(
-          self.jobs, jobs_denylist, jobs_allowlist)
+    jobs_list = []
+
+    if evidence_.config['abort']:
+      jobs_list = [AbortJob]
```

Comment:
We may have talked about this before, but I don't remember the details.  Instead of scheduling this as a normal task, I wonder if we want to just run `state_manager.update_task(task)` on this and completely bypass sending this to the worker.